skyportal/handlers/api/hermes/hermes.py

```diff
@@ -0,0 +1,240 @@
+import functools
+import json
+
+import requests
+
+from baselayer.app.access import auth_or_token
+from baselayer.app.env import load_env
+from baselayer.app.flow import Flow
+from baselayer.app.handlers.base import BaseHandler
+from baselayer.log import make_log
+
+from ....models import TNSRobot
+from ....utils.data_access import get_publishable_obj_photometry
+
+_, cfg = load_env()
```

Comment:
What I would suggest (we do that in some handlers already) is to check at startup (so anywhere in the top level of the file, say here after the imports) if the app is configured to use hermes. You just create a boolean that is set to True if there is all 3 config keys that are required for this to work. If not, then you can have a decorator or just simply an if statement in the handlers that require hermes to be configured to work, and return an error saying "this instance isn't configured to use Hermes, missing ...". That way it's super clear for the user why they can't use it and they can report to the admin, and you don't need to look at the config more than once.

Reply:
Caus like for example I see that you check some error if the endpoint is missing from the config in your `atch_timeout_and_no_endpoint` decorator but that only happens after calling the handler. The config check should happen before calling the method.